src/Google.Api.Gax.Grpc/PageStreamingSync.cs

```diff
@@ -35,135 +37,63 @@ public sealed class PagedEnumerable<TRequest, TResponse, TResource> : IPagedEnum
         /// <param name="apiCall">The API call made each time a page is required.</param>
         /// <param name="request">The initial request.</param>
         /// <param name="callSettings">The settings to apply to each API call.</param>
-        public PagedEnumerable(ApiCall<TRequest, TResponse> apiCall,
-            TRequest request, CallSettings callSettings)
+        public PagedEnumerable(ApiCall<TRequest, TResponse> apiCall, TRequest request, CallSettings callSettings)
         {
-            _pages = new ResponseEnumerable<TRequest, TResponse, TResource>(apiCall, request, callSettings);
-        }
-
-        /// <inheritdoc/>
-        public IResponseEnumerable<TResponse, TResource> AsPages() => _pages;
-
-        /// <inheritdoc/>
-        public IEnumerator<TResource> GetEnumerator() => _pages.SelectMany(page => page).GetEnumerator();
-
-        /// <inheritdoc/>
-        IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
-    }
-
-    /// <summary>
-    /// A sequence of API responses, each containing a page of resources.
-    /// </summary>
-    /// <typeparam name="TRequest">The API request type.</typeparam>
-    /// <typeparam name="TResponse">The API response type.</typeparam>
-    /// <typeparam name="TResource">The resource type contained within the response.</typeparam>
-    internal sealed class ResponseEnumerable<TRequest, TResponse, TResource> : IResponseEnumerable<TResponse, TResource>
-        where TRequest : class, IPageRequest, IMessage<TRequest>
-        where TResponse : class, IPageResponse<TResource>, IMessage<TResponse>
-    {
-        private readonly CallSettings _callSettings;
-        private readonly TRequest _request;
-        private readonly ApiCall<TRequest, TResponse> _apiCall;
-
-        internal ResponseEnumerable(ApiCall<TRequest, TResponse> apiCall, TRequest request, CallSettings callSettings)
-        {
-            _callSettings = callSettings;
-            _request = request;
             _apiCall = apiCall;
+            _request = request;
+            _callSettings = callSettings;
         }
 
-        /// <inheritdoc />
-        public IResponseEnumerator<TResponse> GetEnumerator() =>
-            new ResponseEnumerator(_callSettings, _request.Clone(), _apiCall);
-
-        /// <inheritdoc />
-        IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
-        /// <inheritdoc />
-        IEnumerator<TResponse> IEnumerable<TResponse>.GetEnumerator() => GetEnumerator();
-
-        /// <inheritdoc />
-        public IEnumerable<FixedSizePage<TResource>> WithFixedSize(int pageSize)
+        /// <inheritdoc/>
+        public IEnumerable<TResponse> AsResponses()
```

Comment:
No it doesn't. It just means that when you start iterating over the result, you'll get back responses rather than the resources.